website/components/layouts/Header/Header.tsx

```diff
@@ -6,6 +6,79 @@ import clsx from 'clsx'
 import SEO from '~/components/SEO'
 import ToggleTheme from '~/components/ToggleTheme'
 
+// 语言选择器组件
+function LanguageSelector() {
+  const [isOpen, setIsOpen] = useState(false)
+
+  useEffect(() => {
+    function handleClickOutside(event: MouseEvent) {
+      const target = event.target as Element
+      if (isOpen && !target.closest('.language-selector')) {
+        setIsOpen(false)
+      }
+    }
+
+    if (isOpen) {
+      document.addEventListener('mousedown', handleClickOutside)
+    }
+
+    return () => {
+      document.removeEventListener('mousedown', handleClickOutside)
+    }
+  }, [isOpen])
+
+  return (
+    <div className="relative language-selector mr-6">
+      <button
+        type="button"
+        className="flex items-center text-gray-700 dark:text-gray-200 hover:text-sky-500 dark:hover:text-sky-400 transition-colors"
+        onClick={() => setIsOpen(!isOpen)}
+      >
+        <span className="mr-1">🌐</span>
+        <span>Language</span>
+        <svg
+          className={clsx(
+            'ml-1 h-4 w-4 transition-transform',
+            isOpen ? 'rotate-180' : ''
+          )}
+          fill="none"
+          viewBox="0 0 24 24"
+          stroke="currentColor"
+        >
+          <path
+            strokeLinecap="round"
+            strokeLinejoin="round"
+            strokeWidth={2}
+            d="M19 9l-7 7-7-7"
+          />
+        </svg>
+      </button>
+      
+      {isOpen && (
+        <div className="absolute right-0 mt-2 w-48 bg-white dark:bg-gray-800 rounded-md shadow-lg py-1 z-50 border border-gray-200 dark:border-gray-700">
+          <a
+            href="http://zh.valtio.dev"
```

Comment:
I prefer to have it external:
```suggestion
            href="https://valtio-nine.vercel.app/"
```